app/views/templates/search/search_object.rb

```diff
@@ -276,20 +289,20 @@ def content
               ]
             }),
             properties: data_section( {
-              :title => 'site.object.meta-label.properties',
-              :sections  => [
+              title: 'site.object.meta-label.properties',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.